source/direct_debit/index.html.md.erb

```diff
@@ -5,248 +5,126 @@ weight: 80
+- direct the user to the `next_url` from the API response
+- store the `mandate_id` from the API response, so you can use it to collect payments against the mandate later
 
-#### 2. Your service directs the user to the `next_url`
+Refer to the <a
+href="https://govukpay-api-browser.cloudapps.digital/#newpayment"
+target="blank">GOV.UK Pay API browser</a> or the <a href="https://github.com/alphagov/pay-publicapi/blob/master/swagger/swagger.json" target="blank">Swagger file</a> for more
+information on the API response.
 
-#### 3. The user provides their account details
+### 3. The user provides their bank account details
 
-For example, account number and sort code for Direct Debit.
+When your service redirects the user to `next_url`, they’ll see a **Set up a Direct Debit** page. The user enters their bank account details and selects **Continue**.
 
-GOV.UK Pay makes an API call to our Direct Debit supplier to verify account
+GOV.UK Pay makes an API call to GoCardless to verify the user's account
 details.
 
-#### 4. GOV.UK Pay presents the user with a confirmation screen
+### 4. GOV.UK Pay presents the user with a confirmation screen
 
 The screen presents the user’s account details. The user grants their consent
 by selecting __Confirm__.  GOV.UK Pay sends an email to the user to confirm
 that their mandate has been set up. If the mandate subsequently fails, the
 GOV.UK Pay team will send an email to update the user that their mandate could
 not be set up.
 
-#### 5. GOV.UK Pay securely passes the user's details on to the Direct Debit provider
+### 5. GOV.UK Pay securely passes the user's details on to the Direct Debit provider
 
 The Direct Debit provider submits a request to BACS to set up a mandate, and
 sends a mandate ID to GOV.UK Pay. At this point, services can send a payment
 request.
 
-#### 6. GOV.UK Pay returns the user to your service using the `return_url`
+### 6. GOV.UK Pay returns the user to your service using the `return_url`
+
+### 7. Call the API to check the status of the mandate
+
+API call: `GET /v1/directdebit/mandates/<mandateid>`
+
+This is to display an appropriate screen to the user.
 
-#### 7. Call the API to check the status of the agreement
+The API response includes a `state` object. This includes the following parameters:
 
-API call: `GET /v1/agreements/<AGREEMENT-ID>`
+- [`status`](#mandate-statuses)
+- [`details`](https://developer.gocardless.com/api-reference/#events-mandate-causes), which explains the reason for the `status`
 
-This is to display an appropriate screen to the user. For a Direct Debit
-payment, this will initially be in a ‘pending’ state until the bank confirms
-the mandate. You can confirm this state by signing into the [GOV.UK Pay admin tool](https://selfservice.payments.service.gov.uk/). The mandate confirmation process takes 2 working days.
+The `status` will initially be in a `pending` state until the bank confirms the mandate. The mandate confirmation process takes 2 working days.
```

Comment:
Not sure about “initially” — initially it will be `created`: it’s only `pending` after the user has provided their details.

Reply:
Thanks - I've updated this in https://github.com/alphagov/pay-tech-docs/pull/298/commits/562c05d6cc099a67190a4bfe298a6acc6291794c.